uber/templates/fields/attendee.html

```diff
@@ -337,22 +337,6 @@ <h2 class="text-center">{{ c.DEALER_APP_TERM|title }}</h2>
             $("#badge_message").text("");
         }
     };
-  var setAmountPaid = function (focus) {
```

Comment:
We might want to replace this with a server-side function, but I can't think of cases where we actually definitely want a new attendee to automatically be recorded as paid.

Reply:
Yeah, just leaving it out for now seems fine until we actually have a need for this use case.